crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -329,19 +329,30 @@ fn extract_descriptions_from_module(module: wasmtime::Module) -> anyhow::Result<
 }
 
 struct WasmCtx {
-    mem: Option<Mem>,
+    mem: Option<wasmtime::Memory>,
```

Comment:
I wouldn't advocate for copying the definition of `Mem` etc from `core`.. but I _would_ advocate for sharing them in e.g. a `memutils` crate or something.

I don't feel strongly about it though. I'm not really equipped to own this part of the CLI either way so :shrug: If you prefer it this way, we can just call it good.

Reply:
It's like 10 lines of code in total, so I feel confident that it doesn't make a difference - reading an LE u32 from linear memory is an operation that's never going to have to change.

Reply:
cool, I'll resolve this then